src/einsteinpy/tests/test_geodesics.py

```diff
@@ -44,7 +44,7 @@ def test_Geodesics_str_returns_members(dummy_data):
     geo = Geodesic(body, time=t, end_lambda=end_lambda, step_size=stepsize)
     assert (
         geo.__str__()
-        == "body name= (obj) , metric=(Schwarzschild) , parent name=(attractor) , parent mass=(6e+24 kg)"
+        == "Geodesic object body name= (obj) , metric=(Schwarzschild) , parent name=(attractor) , parent mass=(6e+24 kg)"
```

Comment:
looks like this change got merged too. It is part of isssue #410.

Reply:
Alright, it seems fine to me. Will see tomorrow what issue needs to be closed.